noxfile.py

```diff
@@ -106,7 +106,7 @@ def cover(session):
     session.run("coverage", "erase")
 
 
-@nox.session(python=DEFAULT_PYTHON_VERSION)
+@nox.session(python="3.8")
 def docs(session):
```

Comment:
Could you bump the sphinx version to 4.0.1?

Reply:
2b549a1